packages/imtr/src/fetch.ts

```diff
@@ -1,26 +1,25 @@
-import { emptyDir, exists, join, makeRunWithLimit } from './deps.ts';
+import { emptyDir, exists, join } from "./deps.ts";
 import type { ActivitiesResponse, APIConfig, TopicInputType } from "./types.ts";
 import { writeJson } from "./util.ts";
 
 type Props = {
   config: string;
   dir: string;
   maxConnections: number;
-}
+};
 
-const keyName = 'STTR_BUILDER_API_KEY';
+const keyName = "STTR_BUILDER_API_KEY";
 
 export default async (argv: Props) => {
-
   const baseDir = join(Deno.cwd(), argv.dir);
   const publicDir = join(baseDir, "public");
   const key = Deno.env.get(keyName);
 
   // dir should exist
-  if (!await exists(baseDir)) {
+  if (!(await exists(baseDir))) {
     return new Error(`directory '${baseDir}' not found`);
   }
-  if (!await exists(publicDir)) {
+  if (!(await exists(publicDir))) {
```

Comment:
Why add extra parentheses around the await?

Reply:
That's prettier https://github.com/prettier/prettier/issues/3089